release-engineering/role-handbooks/branch-manager.md

```diff
@@ -292,77 +310,143 @@ The following are some ways to determine if the release process was successful:
+         BAZEL_VERSION: 0.23.2
+      '1.14':
+         CONFIG: '1.14'
+         GO_VERSION: 1.12.5
+         K8S_RELEASE: stable-1.14
+         BAZEL_VERSION: 0.21.0
+      '1.13':
+         CONFIG: '1.13'
+         GO_VERSION: 1.11.5
+         K8S_RELEASE: stable-1.13
+         BAZEL_VERSION: 0.18.1
+
+   # Note that the last block i.e., the `1.12` block was removed
+   ```
+
+4. Create a PR with this change and wait for it to be merged. ([example PR](https://github.com/kubernetes/test-infra/pull/13876))
+
+   **Wait for the `test-infra-push-kubekins-e2e` presubmit to finish (you can [check on prow](https://prow.k8s.io/?job=post-test-infra-push-kubekins-e2e)).**
+
+   **Pull latest from `master` before continuing.**
+
+5. Generate the new release branch jobs:
+
+   ```shell
+   bazel run //experiment:prepare_release_branch
+   ```
+
+6. Search for and update jobs that must be manually bumped
+
+   `grep` for [`manual-release-bump-required`](https://github.com/kubernetes/test-infra/search?q=manual-release-bump-required&unscoped_q=manual-release-bump-required) in [test-infra](https://github.com/kubernetes/test-infra) and duplicate block entries appropriately
+
+7. Update release dashboards in [`testgrid/config.yaml`](https://github.com/kubernetes/test-infra/blob/master/testgrid/config.yaml) ([example commit](https://github.com/kubernetes/test-infra/pull/13882/commits/8aad58b09ae718139b44c70512d2d133d094ed82))
+   - Remove the oldest release `sig-release-<version>-{master,informing,all}` dashboards e.g., `sig-release-1.12-all`
+   - Add dashboards for the current release e.g., `sig-release-1.16-{master,informing,all}`
+
+8. Check for and resolve configuration errors ([example commit](https://github.com/kubernetes/test-infra/pull/13882/commits/867e9ee0329228c6c79399dc67097d0f94814f8f)):
+
+   ```shell
+   bazel test //testgrid/cmd/configurator/...
    ```
 
-1.  Create a PR with this change and wait for it to be merged. Then wait for the `test-infra-push-kubekins-e2e` presubmit to finish (you can [check on prow](https://prow.k8s.io/?job=post-test-infra-push-kubekins-e2e)). Pull latest from `master` before continuing.
+   **Known issues**
+   - [`pull-kubernetes-{dependencies,verify}` jobs need to be manually forked](https://github.com/kubernetes/test-infra/issues/13886) ([example PR](https://github.com/kubernetes/test-infra/pull/13885))
+   - [Several gke/gce/gci jobs are misconfigured for sig-release-x.y-all dashboards](https://github.com/kubernetes/test-infra/issues/13883)
+
+9. Enable/update the `milestoneapplier` plugin configs for the following repos ([example PR](https://github.com/kubernetes/test-infra/pull/13888)):
+    - `kubernetes/enhancements`
+      - Update config for `master` to current milestone e.g., `1.16`
+    - `kubernetes/kubernetes`
+      - Remove config oldest release branch
+      - Add config for the current release branch
+      - Update config for `master` to current milestone e.g., `1.16`
```

Comment:
s/1.16/v1.16 everywhere in this section and

- Instead of `master`, `release-1.16` should be configured to `v1.16` (master would have been configured in the previous release)
- After code freeze is lifted, `master` should be configured to `v1.17`

Reply:
Yep, keeping the reference to `master` for continuity (in case it wasn't done in the previous cycle) e.g., 1.16 is [the first time we're doing it for new release branches](https://github.com/kubernetes/test-infra/pull/13885).

Reply:
Added a note about post-code freeze.